com.unity.shadergraph/Documentation~/Input-Nodes.md

```diff
@@ -90,7 +90,7 @@
 
 ## Texture
 
-|[Cubemap Asset](Cubemap-Asset-Node.md)|[Sample Cubemap](Sample-Cubemap-Node.md)|
+|[Cubemap Asset](Cubemap-Asset-Node.md)|[Sample Cubemap](Sample-Cubemap-Node.md)|[Sample Cubemap](Sample-Reflected-Cubemap-Node.md)|
```

Comment:
I don't understand why a third column was added here. Should it just replace the existing `[Sample Cubemap](Sample-Cubemap-Node.md)`?

Reply:
I just don't know what this stuff was :), will revert